src/map/itemdb.cpp

```diff
@@ -4529,6 +4531,86 @@ void itemdb_reload(void) {
 	mapit_free(iter);
 }
 
+/**
+* Generates an item link string
+* @param data: Item info
+* @return <ITEML> string for the item
+* @author [Cydh]
+**/
+std::string createItemLink( struct item& item ){
+	std::shared_ptr<item_data> data = item_db.find( item.nameid );
+
+	if( data == nullptr ){
+		ShowError( "Tried to create itemlink for unknown item %u.\n", item.nameid );
+		return "Unknown item";
+	}
+
+	struct item_data* id = data.get();
+
+// All these dates are unconfirmed
+#if PACKETVER_MAIN_NUM >= 20200916 || PACKETVER_RE_NUM >= 20200724
+	const std::string start_tag = "<ITEML>";
+	const std::string closing_tag = "</ITEML>";
+#elif PACKETVER >= 20151104
+	const std::string start_tag = "<ITEM>";
+	const std::string closing_tag = "</ITEM>";
+#elif PACKETVER >= 20100000
+	const std::string start_tag = "<ITEMLINK>";
+	const std::string closing_tag = "</ITEMLINK>";
```

Comment:
I partially broke stuff here. I mixed two different features, but dont worry, we will clean it up.